services/movie_session.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(
+    session_date: str = None
+) -> QuerySet:
```

Comment:
Indicate a `QuerySet` of what instances you return in the return type annotation here and in the other places as well